internal/test/go.mod

```diff
@@ -2,19 +2,33 @@ module github.com/openshift/windows-machine-config-bootstrapper/internal/test
 
 go 1.14
 
+replace (
```

Comment:
Please make the commit, `[build] update dependencies`. Add the commands you ran to perform the update.